diff --git a/markup/goldmark/convert.go b/markup/goldmark/convert.go
index ffe9cd45..441774e7 100644
--- a/markup/goldmark/convert.go
+++ b/markup/goldmark/convert.go
@@ -31,6 +31,7 @@ import (
        "github.com/gohugoio/hugo/markup/converter"
        "github.com/gohugoio/hugo/markup/highlight"
        "github.com/gohugoio/hugo/markup/tableofcontents"
+       mathjax "github.com/litao91/goldmark-mathjax"
        "github.com/yuin/goldmark"
        hl "github.com/yuin/goldmark-highlighting"
        "github.com/yuin/goldmark/extension"
@@ -135,6 +136,10 @@ func newMarkdown(pcfg converter.ProviderConfig) goldmark.Markdown {
                extensions = append(extensions, extension.Footnote)
        }

+       if cfg.Extensions.Mathjax {
+               extensions = append(extensions, mathjax.MathJax)
+       }
+
        if cfg.Parser.AutoHeadingID {
                parserOptions = append(parserOptions, parser.WithAutoHeadingID())
        }
diff --git a/markup/goldmark/goldmark_config/config.go b/markup/goldmark/goldmark_config/config.go
index af33e03d..925e324b 100644
--- a/markup/goldmark/goldmark_config/config.go
+++ b/markup/goldmark/goldmark_config/config.go
@@ -30,6 +30,7 @@ var Default = Config{
                Strikethrough:  true,
                Linkify:        true,
                TaskList:       true,
+               Mathjax:        true,
        },
        Renderer: Renderer{
                Unsafe: false,
@@ -58,6 +59,7 @@ type Extensions struct {
        Strikethrough bool
        Linkify       bool
        TaskList      bool
+       Mathjax       bool
 }

 type Renderer struct {
